detect_variants/detect_variants.cwl

```diff
@@ -17,14 +17,14 @@ inputs:
         secondaryFiles: .bai
     interval_list:
         type: File[]
-    strelka_config:
-        type: File
     dbsnp_vcf:
         type: File?
         secondaryFiles: .tbi
     cosmic_vcf:
         type: File?
         secondaryFiles: .tbi
+    strelka_exome_mode:
```

Comment:
agree, this is fine as-is. No need to plan for future tools with exome modes until we need to support them.